modules/op-query-results-name.adoc

```diff
@@ -0,0 +1,81 @@
+results-testing/results/d064ce6e-d851-4b4e-8db4-7605a23671e4  2023-06-29 02:49:45 +0530 IST           2023-06-29 02:49:56 +0530 IST
+----
+
+. To list the names of all records in a result, run the following command:
++
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} <result-name>
+----
++
+.Example command and output
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed
+Name                                                                                                       Type                                    Start                                   Update
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e9c736db-5665-441f-922f-7c1d65c9d621  tekton.dev/v1beta1.TaskRun              2023-06-29 02:49:53 +0530 IST           2023-06-29 02:49:57 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/5de23a76-a12b-3a72-8a6a-4f15a3110a3e  results.tekton.dev/v1alpha2.Log         2023-06-29 02:49:57 +0530 IST           2023-06-29 02:49:57 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/57ce92f9-9bf8-3a0a-aefb-dc20c3e2862d  results.tekton.dev/v1alpha2.Log         2023-06-29 02:50:05 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e9a0c21a-f826-42ab-a9d7-a03bcefed4fd  tekton.dev/v1beta1.TaskRun              2023-06-29 02:49:57 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/04e2fbf2-8653-405f-bc42-a262bcf02bed  tekton.dev/v1beta1.PipelineRun          2023-06-29 02:49:53 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e6eea2f9-ec80-388c-9982-74a018a548e4  results.tekton.dev/v1alpha2.Log         2023-06-29 02:50:05 +0530 IST           2023-06-29 02:50:05 +0530 IST
+----
+
+. To retrieve the YAML manifest for a pipeline run or task run from a record, run the following command:
++
+[source, terminal]
+----
+$ opc results records get --addr ${RESULTS_API} <record-name>   | jq -r .data.value | base64 -d | xargs -0 python3 -c 'import sys, yaml, json; j=json.loads(sys.argv[1]); print(yaml.safe_dump(j))'
+----
++
+.Example command
+[source, terminal]
+----
+$ opc results records get --addr ${RESULTS_API} results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e9c736db-5665-441f-922f-7c1d65c9d621 | jq -r .data.value | base64 -d | xargs -0 python3 -c 'import sys, yaml, json; j=json.loads(sys.argv[1]); print(yaml.safe_dump(j))'
+----
+
+. To retrieve the logging information for a task run from a record, use the log record name. To get the log record name, replace `records` with `logs` in record name.
++
+[source, terminal]
+----
+opc results records get --addr ${RESULTS_API} <log-record-name> | jq -r .data.value | base64 -d
```

Comment:
I don't see value with this command.
```
opc results  logs  get --insecure  --addr ${RESULTS_API} <log-record-name> | jq -r .data | base64 -d
~``

Reply:
--insecure won't apply if the users set up authentication as documented, I believe

Reply:
why?

Reply:
aha the .value - see it now

Reply:
This is what I run daily

Reply:
I don't think that `insecure` is about authentication, rather TLS
> --insecure           determines whether to run insecure GRPC tls request

Reply:
@avinal we describe setting up keys though, it should still not be needed?